### PR TITLE
week1 homework

### PR DESCRIPTION
**Pull request by AnilMula**

### CODE REVIEW

File: week1/task1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿// to reverse a input string
+string input = "world";
+string reversed = ReverseString(input); 
+
+string ReverseString(string input)
+{   var inputLength = input.Length;
```
Comment (calindurnea) [inline week1/task1/Program.cs]: The var here should go on a new line

File: week1/task2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿
+string input = "Intellectualization";
+int vowelCount = GetVowelCount(input.ToLower());
+
+int GetVowelCount(string input)
+{
+    int count = 0;
+    int inputLength = input.Length;
+    for(int i = 0; i < inputLength; i++)
```
Comment (calindurnea) [inline week1/task2/Program.cs]: This can also use a `foreach`

File: week1/task5/Program.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+else
+{
+    int[][] splitArray = Split(input);
+    int[] result = AddArray(splitArray);
+    WriteResult(result);
+}
+
+int[][] Split(int[] input)
+{
+    int[] one = new int[input.Length / 2];
```
Comment (calindurnea) [inline week1/task5/Program.cs]: instead of calling `input.Length / 2` in multiple places, you can store it in a variable and use it where needed

**Review by calindurnea — COMMENTED** [review]
Hello :) 
Really well done! 👍 
I have a couple of comments for you to check out. 
To help with formatting you can also try to use `Shift + Alt + F` in VS code to auto format the code.

**Comment by Vertygo** [comment]
@AnilMula regarding the review comments do you need help?

**Pull request merged**